src/__tests__/canonicalFields.int.test.ts

```diff
@@ -1,14 +1,55 @@
 import request from 'supertest';
 import { app } from '../app';
+import { db } from '../database';
+import type { Result } from 'tinypg';
 
 const agent = request.agent(app);
 
 describe('/canonicalFields', () => {
   describe('/', () => {
-    it('should return HTTP Status Code 200 OK', async () => {
+    it('returns an empty array when no data is present', async () => {
       await agent
         .get('/canonicalFields')
-        .expect(200);
+        .expect(200, []);
```

Comment:
Out of curiosity, I did not notice a difference between chaining the `expect` call like `.expect(...` versus having it as a separate statement `expect(...`. Is there any benefit or drawback to either?

Reply:
So actually it's just running the assertion via two different libraries!

`.expect()` is using the functionality of `supertest` -- which has a pattern of `expect(status code, body)`

`expect()` is using the functionality of `jest` -- which is a whole thing that can't be summed up in one line.

Basically my approach was that whenever I can verify the result within supertest I stuck with that, but when dealing with more complex expectations (e.g. I want to match against a shape of an object rather than a literal value) I had to move to jest.

Reply:
Nice, this makes sense.